job_executor_test.go

```diff
@@ -308,7 +308,7 @@ func TestJobExecutor_Execute(t *testing.T) {
 		require.Len(t, job.Errors, 1)
 		require.WithinDuration(t, time.Now(), job.Errors[0].At, 2*time.Second)
 		require.Equal(t, uint16(1), job.Errors[0].Attempt)
-		require.Equal(t, "throw away this job", job.Errors[0].Error)
+		require.Equal(t, "jobCancelError: throw away this job", job.Errors[0].Error)
```

Comment:
Trying to simplify the "stringifying" code of an errored result caused this to leak in because its added by `jobCancelError`'s `Error()` string. I could change it back to the way it was, but kind of struck me as not the worst thing to be able to easily see that this was a cancellation as opposed to another error?